sonar-php-plugin/src/test/java/org/sonar/plugins/php/PHPSensorTest.java

```diff
@@ -91,10 +92,10 @@
 
   private CheckFactory checkFactory = new CheckFactory(mock(ActiveRules.class));
 
-  private static final Version SONARLINT_DETECTABLE_VERSION = Version.create(6, 7);
-  private static final SonarRuntime SONARLINT_RUNTIME = SonarRuntimeImpl.forSonarLint(SONARLINT_DETECTABLE_VERSION);
-  private static final SonarRuntime NOT_SONARLINT_RUNTIME = SonarRuntimeImpl.forSonarQube(SONARLINT_DETECTABLE_VERSION, SonarQubeSide.SERVER, SonarEdition.COMMUNITY);
-  private static final SonarRuntime SONARQUBE_6_7 = SonarRuntimeImpl.forSonarQube(Version.create(6, 7), SonarQubeSide.SCANNER, SonarEdition.COMMUNITY);
+  private static final Version VERSION_7_9 = Version.create(7, 9);
```

Comment:
Had to update to 7.9 due to missing support of `org.sonarsource.analyzer.commons.ExternalReportProvider::getReportFiles` in 6.7